link/src/main/java/com/stripe/android/link/ui/signup/SignUpViewModel.kt

```diff
@@ -188,8 +192,29 @@ internal class SignUpViewModel @Inject constructor(
         }
     }
 
+    internal class Factory @Inject constructor(
```

Comment:
why do we need this factory? The constructor looks the same as the constructor for the VM itself, so I'm curious why we need the factory instead of just injecting the VM directly

Reply:
You're right. I just pushed a simpler version